modules/slms_3/1.0/slms_3.smk

```diff
@@ -0,0 +1,591 @@
+    "SLMS-3 requires starfish module version 2.0 or higher. "
+)
+
+assert float(CFG_SLMS3["module_versions"]["mutect2"]) >= 2.0, (
+    f"The current mutect2 module version is {CFG_SLMS3['module_versions']['mutect2']}. "
+    "SLMS-3 requires mutect2 module version 2.0 or higher. "
+)
+
+assert float(CFG_SLMS3["module_versions"]["strelka"]) >= 1.1, (
+    f"The current strelka module version is {CFG_SLMS3['module_versions']['strelka']}. "
+    "SLMS-3 requires strelka module version 1.1 or higher. "
+)
+
+# Define rules to be run locally when using a compute cluster
+localrules:
+    _slms_3_input_sage_vcf,
+    _slms_3_input_strelka_vcf,
+    _slms_3_input_mutect_vcf, 
+    _slms_3_input_lofreq_vcf, 
+    _slms_3_mutect2_samples_table, 
+    _slms_3_output_vcf, 
+    _slms_3_all,
+
+##### MODULE CONFIGFILES #####
+
+# Load default module configs and update with values from slms_3 config
+
+for tool in CFG_SLMS3["module_versions"].keys():
+    lcr_modules = config["lcr-modules"]["_shared"]["lcr-modules"] + "modules/"
+    configfile: lcr_modules + "/" +  tool + "/" + CFG_SLMS3["module_versions"][tool] + "/config/default.yaml"
+    snakemake.utils.update_config(
+        config["lcr-modules"][tool], 
+        config["lcr-modules"]["slms_3"][tool]
+    )
+    config["lcr-modules"][tool]["inputs"]["sample_bam"] = CFG_SLMS3["inputs"]["sample_bam"]
+    config["lcr-modules"][tool]["inputs"]["sample_bai"] = CFG_SLMS3["inputs"]["sample_bai"]    
+
+
+##### FIRST PASS VARIANT CALLING MODULE SNAKEFILES #####
+
+lcr_modules = config["lcr-modules"]["_shared"]["lcr-modules"] + "modules/"
+
+# Load Manta first
+include: lcr_modules + "/manta/" + CFG_SLMS3["module_versions"]["manta"] + "/manta.smk"
+
+# Update Strelka config to use Manta Candidate Small Indels    
+config["lcr-modules"]["strelka"]["inputs"]["candidate_small_indels"] = expand(str(rules._manta_output_vcf.output.vcf), vcf_name = "candidateSmallIndels", allow_missing=True)
+
+# Load Strelka, SAGE, and Lofreq
+include: lcr_modules + "/strelka/" + CFG_SLMS3["module_versions"]["strelka"] + "/strelka.smk"
+include: lcr_modules + "/sage/" + CFG_SLMS3["module_versions"]["sage"] + "/sage.smk"
+include: lcr_modules + "/lofreq/" + CFG_SLMS3["module_versions"]["lofreq"] + "/lofreq.smk"
+ 
+##### FIRST PASS VARIANT CALLING RULES #####
+
+
+# Symlinks the input files into the module results directory (under '00-inputs/')
+
+rule _slms_3_input_strelka_vcf: 
+    input:
+        vcf = str(rules._strelka_output_filtered_vcf.output.vcf), 
+        tbi = str(rules._strelka_output_filtered_vcf.output.vcf_tbi)
+    output:
+        vcf = CFG_SLMS3["dirs"]["inputs"] + "vcf/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}.strelka.combined.vcf.gz", 
+        tbi = CFG_SLMS3["dirs"]["inputs"] + "vcf/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}.strelka.combined.vcf.gz.tbi"
+    run:
+        op.relative_symlink(input.vcf, output.vcf)
```

Comment:
I've been advocating for all the input symlinking to change to absolute_symlink

Reply:
Does the fact that this is symlinking the output of another module change your thoughts about this?

Reply:
I suppose it depends on how deep these symlinks go. Would there be any reason not to use in_module=True here?

Reply:
I made these all `in_module=True` because the directories are all set by the module and will be adjacent in the file system.